content/docs/operating/security.md

```diff
@@ -161,3 +161,16 @@ members of the Prometheus development team and the staff of those providers
 have access. If you are concerned about the exact provenance of your binaries,
 it is recommended to build them yourself rather than relying on the
 pre-built binaries provided by the project.
+
+## External audits
+
+[CNCF](https://cncf.io) sponsored an external security audit by
+[cure53](https://cure53.de) which ran from April 2018 to June 2018.
+
+The audit found no major concerns outside of the scope of this document. It
```

Comment:
This is really an unduly generous interpretation of what the report says and omits that the report strongly suggests to change our practices. Also I don't think that this document here would even give people an idea that e.g. the CORS and XSRF vulnerabilities exist and need to be protected against. Those are pretty severe IMO. If an attacker can make me visit a website and then they can either read all my Prometheus data or shut down my Prometheus server, that seems bad, and almost nobody reading this document would anticipate those problems from what is written here.